madgiraffestudios/public/index.html

```diff
@@ -26,7 +26,7 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
-    <title>React App</title>
+    <title>Madgiraffestudios</title>
```

Comment:
my only thing is lets cap the G and S to say MadGiraffeStudios we can address in another PR

Reply:
> my only thing is lets cap the G and S to say MadGiraffeStudios we can address in another PR

Great callout, I agree 👍